### DISCUSSION TITLE
need help: I got "module 'openai' has no attribute 'error'" after following install instruction

### DISCUSSION THREAD

**Discussion by wangztao** [llm-workflow-engine/llm-workflow-engine#331] [Q&A]
1> /login tony
Enter password:
Login successful.

[New Conversation]
(0.7/16384/0): default
tony@gpt-3.5-turbo-1106 1> /ask hi

AttributeError("module 'openai' has no attribute 'error'")
[New Conversation]
(0.7/16384/0): default
tony@gpt-3.5-turbo-1106 1>

**Comment by wangztao** [comment]
is it a know issue? or anything i missed

**Comment by thehunmonkgroup** [comment]
See here: https://github.com/llm-workflow-engine/llm-workflow-engine/issues/329#issuecomment-1814583298
